.gitignore

```diff
@@ -28,3 +28,6 @@ node_modules
 
 # Sqlite
 data.db
+
+# Intellij IDE
+.idea
```

Comment:
You can put this rule into your global .gitignore file, let to keep clean this file :wink: